scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -71,31 +70,6 @@ private val TIMEOUT_ERROR_REGEX = Pattern.compile(
             "ERROR: Processing interrupted: timeout after (?<timeout>\\d+) seconds. \\(File: (?<file>.+)\\)"
 )
 
-// A list of ScanCode-specific LicenseRefs that do not actually name concrete licenses, but which are generic findings
-// that "look like" licenses.
-private val NOASSERTION_LICENSE_REFS = listOf(
-    // https://scancode-licensedb.aboutcode.org/?search=generic
-    "LicenseRef-scancode-agpl-generic-additional-terms",
-    "LicenseRef-scancode-generic-cla",
-    "LicenseRef-scancode-generic-exception",
-    "LicenseRef-scancode-generic-export-compliance",
-    "LicenseRef-scancode-generic-tos",
-    "LicenseRef-scancode-generic-trademark",
-    "LicenseRef-scancode-gpl-generic-additional-terms",
-    "LicenseRef-scancode-patent-disclaimer",
-    "LicenseRef-scancode-warranty-disclaimer",
-
-    // https://scancode-licensedb.aboutcode.org/?search=other
-    "LicenseRef-scancode-other-copyleft",
-    "LicenseRef-scancode-other-permissive",
-
-    // https://scancode-licensedb.aboutcode.org/?search=unknown
-    "LicenseRef-scancode-free-unknown",
-    "LicenseRef-scancode-unknown",
-    "LicenseRef-scancode-unknown-license-reference",
-    "LicenseRef-scancode-unknown-spdx"
-)
-
```

Comment:
I added the previous ScanCode license mappings as defaults, to avoid a breaking change. For FossID this doesn't really make sense, as the licenses are different for every FossID instance.

I think that @sschuberth's proposal makes sense, as it would be more transparent for the user which defaults are being used.

Reply:
@sschuberth I'm glad you reconsider this after #3522 / #3731 :) I would still prefer a `default.conf` over hardcoded values in Kotlin, because I think it's beneficial to the user, but I wouldn't agree that it should replace the `reference.conf`. The purpose of `default.conf` is to provide a meaningful default configuration, while the purpose of `reference.conf` is to showcase all possible configuration options.

Reply:
> The purpose of `default.conf` is to provide a meaningful default configuration, while the purpose of `reference.conf` is to showcase all possible configuration options.

Right, but IMO with `default.conf` there should be no implicit default anymore. So if everything is explicit, `default.conf` basically *is* a `reference.conf`. The only question would be about examples for credentials etc. that should not go into a `default.conf`. Maybe these could be added as comments into `default.conf`, and the existing `ort config --show-default` could simply display them?

Reply:
It's more than just credentials, for example the `default.conf` would likely configure a local file based storage for scan results, but we also need to demonstrate how the other storage types can be configured. Of course this could also be done with comments in `default.conf`, but then our current testing approach wouldn't work. Let's discuss the details once anyone wants to pick this topic up.